vignettes/subtotals.Rmd

```diff
@@ -0,0 +1,149 @@
+---
+title: "Subtotals"
+description: "Modify subtotals and headings from R"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Subtotals}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+```{r load data, include=FALSE}
+## Because the vignette tasks require communicating with a remote host,
+## we do all the work ahead of time and save a workspace, which we load here.
+## We'll then reference saved objects in that as if we had just retrieved them
+## from the server
+library(crunch)
+load("vignettes.RData")
+options(width=120)
+```
+
+A common task in the Market research world is to collapse two or more categories together to see how the collapsed categories compare to one another. For example, if you asked people to rate their preference on a scale of 1-10 you might want to see how the people who provide a rating between 1 and 5 compare to those who rated it between 6 and 10. This goes by a number of names like _Top Box_, _Top Two Box_, or _Nets_. But in Crunch it is called _Subtotals_
+
+Subtotals can be applied to any Categorical Crunch Variables, and we can check if a variable has subtotals with the `subtotals()` function. If it returns `NULL`, there are no subtotals.
```

Comment:
Super super minor, but I would change "Subtotals can be applied to any Categorical Crunch Variables" to "Subtotals can be applied to any Categorical Crunch Variable"